### DISCUSSION TITLE
Application delivery configuration in netbox

### DISCUSSION THREAD

**Discussion by jcralbino** [netbox-community/netbox#8438] [Ideas]
Hello Team

I would like to know how is the community handling the need to document load balancer configuration in netbox.

As netbox becomes the sot for network infrastructure, one key missing part is how to document the load balancer configuration for application delivery. 

Now i am saving the load balancer info as a device in netbox , and using the services to document names and https services associated with the device (vip) but in order to document the actual pool of servers for a given vip is not something easily supported.
The solution we used was to create the information as a local configuration context.
But I would prefer to have a database structure behind it.
Do we any plug-in focused in providing a better structure for this use case? 
@candlerb 

The fhrg object seems to go in the right direction where we are able to group different IPs, from different devices in a VIP.

Is this something we can place in the core roadmap ? 
@jeremystretch

**Comment by candlerb** [comment]
> Do we any plug-in focused in providing a better structure for this use case?

I'm not tracking third-party plugins.  It does sound like something you could write for your use case.

**Reply by jcralbino** [reply]
Thanks for the reply

**Comment by DanSheps** [comment]
I don't believe anything is planned in core.

This would be best handled as a separate plugin, at least for now.
